debian/preinst

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# preinst script for st2
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <new-preinst> `install'
+#        * <new-preinst> `install' <old-version>
+#        * <new-preinst> `upgrade' <old-version>
+#        * <old-preinst> `abort-upgrade' <new-version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+ST2_USER=st2
+
+create_users() {
+  # create st2 user (services user)
+  (id $ST2_USER 1>/dev/null 2>&1) ||
+    adduser --group --disabled-password --no-create-home --system $ST2_USER
+
+  # make st2 member of st2packs group
+  (getent group $PACKS_GROUP 1>/dev/null 2>&1) || groupadd -r $PACKS_GROUP
+  (groups $ST2_USER 2>/dev/null | grep -q "\b${PACKS_GROUP}\b") ||
+    usermod -a -G $PACKS_GROUP $ST2_USER
```

Comment:
This part could be removed starting from the `# make st2 member of st2packs group` as it's not needed for st2chatops.

Reply:
👍